app/views/team/admin.scala

```diff
@@ -178,13 +178,7 @@ $('#form3-message').val($('#form3-message').val() + e.target.dataset.copyurl + '
               case (remaining, until) =>
                 frag(
                   p(cls := (remaining <= 0).option("error"))(
-                    "You can send up to ",
-                    lila.app.mashup.TeamInfo.pmAllCredits,
-                    " team messages per week. ",
-                    strong(remaining),
-                    " messages remaining until ",
-                    momentFromNowOnce(until),
-                    "."
+                    teamMessagesPerWeekLimit(lila.app.mashup.TeamInfo.pmAllCredits, strong(remaining), momentFromNowOnce(until))
```

Comment:
have you tested this code, are the singular and plural form correctly selected? I would guess they aren't

Reply:
oh